TYPO3.Flow/Configuration/Settings.yaml

```diff
@@ -168,7 +168,10 @@ TYPO3:
 
         'preprocess':
           position: 'before process'
-          chain: []
+          chain:
+            'trustedProxies':
+              position: 'start'
+              component: 'TYPO3\Flow\Http\Component\TrustedProxiesComponent'
```

Comment:
It's currently needed for b/c because it provides some functionality that was previously in the Request (overriding host/port from headers, getting client IP address).